### ISSUE TITLE
This library requires legacy openssl support

### ISSUE THREAD

**Issue by pereriksson** [anyfin/bankid#60]
BankID certificates are encrypted using the RC2-40-CBC algorithm, which was probably dropped in OpenSSL 3.

As Node 17 switched to openssl 3, and bankid certificates use the RC2 algorithm, a node application using this library must be started with `node-options="--openssl-legacy-provider"`.

I think an average developer expects this library to work out-of the box with the certificate they get from BankID.

Maybe we should mention this option is required in our README?

We use axios, which uses follow-redirects, so the symptom the developer will experience is:
```
 Error: unsupported
    at configSecureContext (node:internal/tls/secure-context:278:15)
    at Object.createSecureContext (node:_tls_common:116:3)
    at Object.connect (node:_tls_wrap:1718:48)
    at Agent.createConnection (node:https:169:22)
    at Agent.createSocket (node:_http_agent:342:26)
    at Agent.addRequest (node:_http_agent:289:10)
    at new ClientRequest (node:_http_client:337:16)
    at Object.request (node:https:377:10)
    at RedirectableRequest._performRequest (/app/node_modules/bankid/node_modules/follow-redirects/index.js:273:24)
    at new RedirectableRequest (/app/node_modules/bankid/node_modules/follow-redirects/index.js:61:8)
```

**Comment by elitan** [comment]
This was fixed here: https://github.com/anyfin/bankid/pull/59

**Comment by pereriksson** [comment]
> This was fixed here: #59

Thanks but this is certificate-related in production. I have created a PR with the rationale and solution here.
https://github.com/anyfin/bankid/pull/61

**Comment by pmmmwh** [comment]
Fixed in #62
